terraform/environments/cooker/macie.tf

```diff
@@ -0,0 +1,32 @@
+
+# Create macie account
```

Comment:
I don't think these comments are necessary - the resource is pretty self explanatory and attributes are explained in the terraform documentation.

Reply:
Comments were just for clarity. They are shown in the documentation so I will remove them from here (apart from the create macie account and create job ones